task8-main/src/main/java/com/example/classloader/app/Demo.java

```diff
@@ -0,0 +1,82 @@
+package com.example.classloader.app;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.*;
+import java.lang.reflect.Constructor;
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Method;
+import java.net.URL;
+import java.net.URLClassLoader;
+import java.util.*;
+
+public class Demo {
+    private static final Logger LOGGER = LoggerFactory.getLogger(Demo.class);
+
+    public static void main(String[] args) throws IOException, ClassNotFoundException, NoSuchMethodException, InstantiationException, IllegalAccessException, InvocationTargetException {
+
+        String pathName = getModulsLocation("location.properties");
+        List<File> jars = Arrays.asList(new File(pathName).listFiles());
+        URL[] urls = new URL[jars.size()];
+        for (int i = 0; i < jars.size(); i++) {
+            try {
+                urls[i] = jars.get(i).toURI().toURL();
+                System.out.println(urls[i]);
+            } catch (Exception e) {
+                e.printStackTrace();
+            }
+        }
+        URLClassLoader childClassLoader = new URLClassLoader(urls, ClassLoader.getSystemClassLoader());
+        Thread.currentThread().setContextClassLoader(childClassLoader);
```

Comment:
with this line you are replacing existing class loader with another one. So, if you have some other libraries from other paths, that may be lost... you need to add class path